iron-dropdown.html

```diff
@@ -99,6 +99,26 @@
           },
 
           /**
+           * A pixel value that will be added to the position calculated for the
```

Comment:
super nitty, but I would rephrase this as "A pixel value that will be added to the horizontal position of the dropdown, in the direction given by `horizontalAlign`". Or something like that (too much?)